src/sbomnix/sbomdb.py

```diff
@@ -383,6 +383,21 @@ def _drv_to_cdx_component(drv, uid="store_path"):
     if "meta_description" in drv._asdict() and drv.meta_description:
         component["description"] = drv.meta_description
     _cdx_component_add_licenses(component, drv)
+    if drv.patches:
+        security_patches = []
+        for p in drv.patches.split(" "):
+            print(p)
+            m = re.search(r"CVE-\d{4}-\d+", p)
```

Comment:
Unless there's a good reason not to, I would ignre the case in this match in case the data is dirty.